source/api_documentation/network_integration/index.rst

```diff
@@ -31,6 +31,7 @@ The API uses the following terms and their aliases:
    advertiser_affiliate_relationships/index
    networks/index
    promo_numbers/index
+   call_activity_tracking/index
```

Comment:
My only thought with this is.. looks like you are still sharing/using the existing `advertiser_campaigns` endpoint within the Network Integration API family. 

Was it already discussed whether to add this to the existing `advertiser_campaigns/index` versus having a separate page? 

I feel like if we keep it separate we should at least call out that it is the same endpiont and to refer to `<link to advertiser_campaigns/index>` for more information on the URL structure and required params/structure of the request body.

Reply:
@omeedrabani Can you add your perspective to this comment when you have a moment (I know you're playing catch-up today)?  I was following the pattern we had established for `call_recording_condition`, but happy to re-work if it makes sense.

Reply:
> My only thought with this is.. looks like you are still sharing/using the existing advertiser_campaigns endpoint within the Network Integration API family.

This is expected.

From what I remember, `call_recording_condition` was made its own page so it's easier to find/read because it has a lot of info and the `advertiser_campaigns/index` page already is a really large page.

Given that the info for `call_activity_tracking` isn't as large, I don't have a strong opinion here about whether it should be its own page or not.

> I feel like if we keep it separate we should at least call out that it is the same endpiont and to refer to <link to advertiser_campaigns/index> for more information on the URL structure and required params/structure of the request body.

This sounds good, assuming we keep it as its own page.

@nburwell Any thoughts?

Reply:
If there is already precedence for this approach, sounds good to me to keep how it is then (as a separate page). 

I would like to see some sentence and link back to the main `advertiser_campaigns/index` to provide the context of what API endpoint this is.

Reply:
@omeedrabani Would you be so kind as to provide the verbiage you'd like to see, in that case? Something to the effect of this?

> `<link to advertiser_campaigns/index>`Learn more about the Network Integration API family`</link>`, its URL structure, and required parameters of the request body.

Reply:
Maybe something like 

> For full details on creating and updating campaigns via the API, see [Advertiser Campaigns](`<link to advertiser_campaigns/index>`).

Reply:
Here's [a link to that change](https://github.com/Invoca/developer-docs/pull/271/commits/f828fd8e62ed5ea807bb03003e65785018068644) if you'd like to have a look. Thanks!